response_operations_ui/controllers/party_controller.py

```diff
@@ -160,6 +161,8 @@ def update_contact_details(respondent_id, form, ru_ref='NOT DEFINED'):
     }
 
     old_contact_details = get_respondent_by_party_id(respondent_id)
+    logger.info("new details", new_details=new_contact_details)
```

Comment:
We shouldn't be logging either of these lines - personal info can't go into logs